Source/WTF/wtf/cf/URLCF.cpp

```diff
@@ -73,18 +73,15 @@ RetainPtr<CFURLRef> URL::createCFURL() const
     return result;
 }
 
+#if !PLATFORM(WIN)
```

Comment:
This should not be needed because this file is not compiled on `PLATFORM(WIN)` any more.